src/webhint-theme/source/core/css/about.css

```diff
@@ -34,7 +34,7 @@ h2.changelog__date {
     font-size: 2.4rem;
     font-weight: 500;
     color: #4700a3;
-    text-transform: uppercase;
+    text-transform: capitalize;
```

Comment:
Will `text-transform: capitalize` work across all locales? I honestly have no idea, though I doubt it has as many problems as `uppercase`. If in doubt it may be safest to handle the capitalization as part of our localization process...

Reply:
From MDN: 

The text-transform property is not reliable for some locales; for example, text-transform: uppercase won't work properly with languages such as Irish/Gaelic. For example, App Size in English may be capitalized via text-transform: uppercase to APP SIZE but in Gaelic this would change Meud na h-aplacaid to MEUD NA H-APLACAID which violates the locales orthographic rules, as it ought to be MEUD NA hAPLACAID. In general, localizers should make the decision about capitalization. If you want to display WARNING, add a string with that capitalization, and explain it in the localization note.

https://developer.mozilla.org/en-US/docs/Mozilla/Localization/Localization_content_best_practices

I think the biggest concern is all uppercase, as far as I can tell.

Reply:
Ok, thanks for the follow-up!